aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -627,26 +886,31 @@ module aptos_framework::coin {
     public fun withdraw<CoinType>(
         account: &signer,
         amount: u64,
-    ): Coin<CoinType> acquires CoinStore {
+    ): Coin<CoinType> acquires CoinStore, CoinConversionMap, CoinInfo, PairedCoinType {
         let account_addr = signer::address_of(account);
-        assert!(
-            is_account_registered<CoinType>(account_addr),
-            error::not_found(ECOIN_STORE_NOT_PUBLISHED),
-        );
-
-        let coin_store = borrow_global_mut<CoinStore<CoinType>>(account_addr);
-        assert!(
-            !coin_store.frozen,
-            error::permission_denied(EFROZEN),
-        );
-
-        event::emit_event<WithdrawEvent>(
-            &mut coin_store.withdraw_events,
-            WithdrawEvent { amount },
-        );
-        event::emit(Withdraw<CoinType> { account: account_addr, amount });
-
-        extract(&mut coin_store.coin, amount)
+        if (is_account_registered<CoinType>(account_addr)) {
```

Comment:
This will likely break when used with balance. Balance sums up both coin and FA balances but this only takes from either